test/sql/updates/post.compression.sql

```diff
@@ -14,3 +14,18 @@ INNER JOIN _timescaledb_catalog.hypertable hypertable ON (chunk.hypertable_id =
 WHERE hypertable.table_name = 'compress' and chunk.compressed_chunk_id IS NULL;
 
 SELECT * FROM compress ORDER BY time DESC, small_cardinality;
+
+ANALYZE compress;
+select random(),count(*) from compress;
```

Comment:
What is the idea behind using random here?

Reply:
this was only temporary to debug the update test